### ISSUE TITLE
when I upgrade to version 1.2.0 , i have the problem : "Uncaught Error: Unexpected value 'RoundProgressModule' imported by the module 'SharedModule'. Please add a @NgModule annotation." . How to fix ?

### ISSUE THREAD

**Issue by jerry7411** [crisbeto/angular-svg-round-progressbar#143]
I upgrade my angular verion 4.4.6 to 5.0.0 and upgrade angular-svg-round-progressbar version 1.1.1 to 1.2.0 then i have the problem : ": "Uncaught Error: Unexpected value 'RoundProgressModule' imported by the module 'SharedModule'. Please add a @NgModule annotation." .  How to fix ?

**Comment by CarlosTorrecillas** [comment]
Looks like is the same as https://github.com/crisbeto/angular-svg-round-progressbar/issues/142

**Comment by papaiatis** [comment]
I have the same problem here!

**Comment by maku** [comment]
Same here

**Comment by crisbeto** [comment]
The demo is set up using Angular 5 and it seems to work. How are you compiling your app? AFAIK as of Angular 5 you should use the `@angular/compiler-cli`.

**Comment by CarlosTorrecillas** [comment]
My tsconfig.app.json:

{
  "extends": "../tsconfig.json",
  "compilerOptions": {
    "outDir": "../out-tsc/app",
    "baseUrl": "./",
    "module": "es2015",
    "types": ["node"]
  },
  "exclude": [
    "test.ts",
    "**/*.spec.ts"
  ]
}

My tsconfig.json:
dom"
    ]
  }
}

The only thing I took from v5.0.0 is the preserve white spaces option = false which I added to the configuration

**Comment by maku** [comment]
@crisbeto I made a fresh project with angular-cli (1.5) and I tried also stackblitz.com (https://stackblitz.com/edit/angular-cxhpto) with the simplest possible test. 

In both cases I got 

```
Error: Unexpected value 'RoundProgressModule' imported by the module 'AppModule'. Please add a @NgModule annotation.
```

**Comment by jerry7411** [comment]
@crisbeto i have @angular/compiler-cli in my angular . it is version 5.0.0 . I don't think that is the problem

**Comment by m1nd53t** [comment]
I had the same problem, for me the solution was just copying the src folder into my instead of loading it from yarn. I did remove the tsconfig from the source file, don't know if that matters.

I think that there is a problem with the tsconfig settings for distributing as a package because the src works.

**Comment by ianlintner-wf** [comment]
npm install angular-svg-round-progressbar@1.1.0 --save

**Comment by mdybich** [comment]
Same problem here

**Comment by BOb511** [comment]
So, I looked around and found this: https://stackoverflow.com/questions/45468663

Once i removed the node_modules/angular-svg-round-progressbar/**node_modules** directory both `ng serve` and `ng build` worked again. (Angular 5.0.0, progressbar 1.1.1)

**Comment by sharok** [comment]
The same error, I migrated my project to Angular 5.0.3 and updated the lib to 1.2.0. Did clean npm install. Didn't help.

**Comment by crisbeto** [comment]
Should be fixed in the 1.2.1 release. Let me know if it still doesn't work and I'll reopen the issue.
